package.json

```diff
@@ -38,12 +39,14 @@
 		"execa": "^8.0.1",
 		"fs-fixture": "^2.4.0",
 		"get-node": "^15.0.1",
+		"glob-to-regexp": "^0.4.1",
```

Comment:
Noticed that this dependency is marked as archived/read-only, while a more popular alternative https://github.com/isaacs/minimatch is still actively maintained (and apparently used by npm internals).

Was there some reason you chose glob-to-regex instead?